data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/schema/Peer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title" : "Peer",
+  "type" : "object",
+  "required" : [ "peer_id", "address", "state", "direction" ],
+  "properties" : {
+    "peer_id" : {
+      "type" : "string",
+      "description" : "Cryptographic hash of a peer’s public key. '[Read more](https://docs.libp2p.io/concepts/peer-id/)",
+      "example" : "QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N"
+    },
+    "enr" : {
+      "type" : "string",
+      "description" : "Ethereum node record. Not currently populated. [Read more](https://eips.ethereum.org/EIPS/eip-778)",
+      "example" : "example: enr:-IS4QHCYrYZbAKWCBRlAy5zzaDZXJBGkcnh4MHcBFZntXNFrdvJjX04jRzjzCBOonrkTfj499SZuOh8R33Ls8RRcy5wBgmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQPKY0yuDUmstAHYpMa2_oxVtw0RW_QAdpzBQA8yWM0xOIN1ZHCCdl8"
+    },
+    "address" : {
+      "type" : "string",
+      "description" : "[Read more](https://docs.libp2p.io/reference/glossary/#multiaddr)",
+      "example" : "/ip4/7.7.7.7/tcp/4242/p2p/QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N"
+    },
+    "state" : {
+      "$ref" : "#/components/schemas/State"
+    },
+    "direction" : {
+      "$ref" : "#/components/schemas/Direction"
```

Comment:
hrm this didnt come out as a definition, so must already be used somewhere, check its what we mean, and it's also an enumeration
`Enum: "inbound" "outbound"`